.github/CHANGELOG.md

```diff
@@ -134,6 +134,9 @@
 * Remove `pennylane.ops.op_math.controlled_decompositions.ctrl_decomp_zyz` tests with `len(control_wires) > 1`.
   [(#757)](https://github.com/PennyLaneAI/pennylane-lightning/pull/757)
 
+* Add support for Scipy v1.14.
+  [(#776)](https://github.com/PennyLaneAI/pennylane-lightning/pull/776)
```

Comment:
I assume we still test against <1.14 too in the CI, so that these changes still work with older versions?

Reply:
Seems like the CI pulled 1.13.

Reply:
yes! we use < 1.13 in Catalyst and the comp checks pass against this PR.